cli/config/configuration-fc.toml.in

```diff
@@ -250,6 +250,14 @@ path = "@SHIMPATH@"
 #trace_mode = "dynamic"
 #trace_type = "isolated"
 
+# Comma separated list of kernel modules and their parameters.
+# These modules will be loaded in the guest kernel using modprobe(8).
+# The following example can be used to load two kernel modules with parameters
+#  - kernel_modules=["e1000e InterruptThrottleRate=3000,3000,3000 EEE=1", "i915 enable_ppgtt=0"]
+# The first word is considered as the module name and the rest as its parameters.
+#
+kernel_modules=[]
```

Comment:
This obviously relies on the admin specifying a set of modules that is going to be available in the guests kernel. As such, I think it is worth stating here what happens if:

- the module is not available in the guest kernel.
- `modprobe` disabled in the guest environment.
- the module is architecture-specific and the guest is running an incompatible architecture.